### PR TITLE
First PR and test tasks

### PR DESCRIPTION
**Pull request by upworkroozbeh**

### CODE REVIEW

File: upwork-devs/Roozbeh Moayydeian/GW.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- private and limited access storage on S3 or Azure SCS
+- keeping everything isolated and private by restricted access.
+
+- step 1: building image and pushing to ECR/ACR
+	docker build -t {REPO_NAME} .
+	docker push {REPO_NAME}
+
+- Creating CI/CD pipeline based on files repo or whatever we have for starting the pipeline (S3 uploads, SNSQ, ...).
+- Creating one k8s job per file. jobs automatically removed after processing files and only logs can be accessible for limited/unlimited time (https://kubernetes.io/docs/concepts/workloads/controllers/job/).
+- Storing results on S3/Azure SCS
```
Comment (DinisCruz) [inline upwork-devs/Roozbeh Moayydeian/GW.md:27]: actually, we also need to have just about all of this in our local and test environment :)

File: upwork-devs/Roozbeh Moayydeian/glasswall-k8s-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        name: glasswall-test
+        envFrom:
+        - configMapRef:
+            name: glasswall-test-configmap
+        imagePullPolicy: Always
+        volumeMounts:
+         - name: localtime-vol
+           mountPath: /etc/localtime
+         - name: timezone-vol
+           mountPath: /etc/timezone
```
Comment (DinisCruz) [inline upwork-devs/Roozbeh Moayydeian/glasswall-k8s-deployment.yaml:30]: this is interesting, you are mounting the local time and local time zone :)

really cool technique

File: upwork-devs/Roozbeh Moayydeian/run-engine.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+docker run -it --rm -v ${PWD}/input:/input -v ${PWD}/output:/output glasswall-rebuild-eval
```
Comment (DinisCruz) [inline upwork-devs/Roozbeh Moayydeian/run-engine.sh:1]: now that you know how to use the GW engine :)
next step is to do this from a k8s job

**Review by DinisCruz — APPROVED** [review]
Nice first commit

Can you remove the output files from the commit?

**Review by dinis-cruz-gw — APPROVED** [review]

**Pull request merged**